### DISCUSSION TITLE
Opening a github link in a new tab does not respect dark mode

### DISCUSSION THREAD

**Discussion by jwtracy** [community/community#120154] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I have github appearance set to track my Mac system setting. When I `Command-Click` a link or right click and `Open in new tab` the new web page does not respect dark mode at all. Staying on this page for a while and browsing links as normal still does not use dark mode.

Steps to reproduce:
1. appearance traces system settings which are in dark mode on mac
2. navigate to a PR in dark mode.
3. open a check run link in a new tab
4. see new tab is not in dark mode

**Comment by ebndev** [comment]
Hey @jwtracy, thanks for being a part of the GitHub community!

I'm curious if other folks have had this problem. 

In the meantime maybe you could try selecting "Single Theme" and pick a dark theme that way. If you're still encountering the same issue, it might be because of another reason.

**Comment by travi** [comment]
i've been seeing similar behavior, although i dont need to open a new tab to trigger the behavior. it seems to happen when loading an actions log from a link (which would be the new tab scenario) or when performing other actions, like restarting a build, from the actions results pages. after it switches to light mode, i have to navigate away from the workflow results pages and refresh the browser before dark mode returns.

i've asked other team members if they have noticed similar switches from dark to light mode, but have not found others seeing the same behavior. this makes me wonder if the situation is limited to something more specific than just light vs dark. i have theme mode set to "sync with system" and use "Light Protanopia & Deuteranopia" and "Dark Protanopia & Deuteranopia" in case those details are related.
